### DISCUSSION TITLE
[Bug] Why logs (use logback toolkit) TraceId lost in ActiveMQConsumer Method

### DISCUSSION THREAD

**Discussion by stardyun** [apache/skywalking#8427] [Q&A]
### Search before asking

- [X] I had searched in the [issues](https://github.com/apache/skywalking/issues?q=is%3Aissue) and found no similar issues.

### Apache SkyWalking Component

Java Agent (apache/skywalking-java)

### What happened

1. config logback.xml 
```
add org.apache.skywalking.apm.toolkit.log.logback.v1.x.TraceIdPatternLogbackLayout
add %tid in pattern
```

2. log in ActiveMQConsumer Method
```
@JmsListener(destination = "LOAN_CONTRACT_AUTO_SIGN")
public void receive(String data) {
    log.info("loan_contract_auto_sign, data:{}", data);
}
```

`[ifs] 2022-01-13 15:35:11,536 INFO [DefaultMessageListenerContainer-1] [TID:N/A] [user: unknown] loan_contract_auto_sign, data: ...`

3.  **tid is N/A**

### What you expected to happen

example:
```
@JmsListener(destination = "LOAN_CONTRACT_AUTO_SIGN")
public void receive(String data) {
    log.info("loan_contract_auto_sign, data:{}", data);
}
```

`[ifs] 2022-01-13 15:35:11,536 INFO [DefaultMessageListenerContainer-1] [TID:N/A] [user: unknown] loan_contract_auto_sign, data: ...`

**_N/A There are normal_**

### How to reproduce

example:
```
@JmsListener(destination = "LOAN_CONTRACT_AUTO_SIGN")
public void receive(String data) {
    log.info("loan_contract_auto_sign, data:{}", data);
}
```

`[ifs] 2022-01-13 15:35:11,536 INFO [DefaultMessageListenerContainer-1] [TID:N/A] [user: unknown] loan_contract_auto_sign, data: ...`

### Are you willing to submit PR?

- [ ] Yes I am willing to submit a PR!

### Code of Conduct

- [X] I agree to follow this project's [Code of Conduct](https://www.apache.org/foundation/policies/conduct)

**Comment by wu-sheng** [comment]
The ActiveMQ case could be found here, https://github.com/apache/skywalking-java/tree/main/test/plugin/scenarios/activemq-scenario
